### PR TITLE
realtek: add support for Panasonic Switch-M8eG PN28080K

### PR DESCRIPTION
**Pull request by musashino205**
- enable pca953x driver in realtek target
  - to use the LED under the PCA9539PW for system status of OpenWrt while booting
- add support for Panasonic Switch-M8eG PN28080K

### CODE REVIEW

File: target/linux/realtek/dts/rtl8380_panasonic_m8eg-pn28080k.dts [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+
+		full_duplex {
+			label = "green:full";
+			gpios = <&gpio2 10 GPIO_ACTIVE_LOW>;
+		};
+
+		loop_history {
+			label = "green:loop_history";
+			gpios = <&gpio2 11 GPIO_ACTIVE_LOW>;
+		};
```
Comment (adschm) [inline target/linux/realtek/dts/rtl8380_panasonic_m8eg-pn28080k.dts:64]: As always, I'm not really a fan of mixing different "names" for node and label here. If it's full duplex, why not call the led "green:full-duplex" as well so the name will actually explain what it's for?

Why is the label status_eco and the node just "status". And what's any_col? ...

Comment (musashino205) [reply target/linux/realtek/dts/rtl8380_panasonic_m8eg-pn28080k.dts:64]: @adschm I used the generic name for node name, and for label, I used the name that is actually printed.

IMG_20210529_103801

> Why is the label status_eco and the node just "status"

It's my bad, I'll change the node name to ```status_eco```.

> what's any_col

If this mode is selected, the port LED where the collision has occurred so far will be turned on.

**Comment by musashino205** [comment]
UPDATE:

- update the node names and dt-labels
  - ```status_(green|amber)``` -> ```status_eco_(green|amber)```
    - dt-label: ```led_status_(green|amber)``` -> ```led_status_eco_(green|amber)```
  - ```collision``` -> ```any_collision```

**Comment by predators46** [comment]
Wao

File: target/linux/realtek/dts/rtl8380_panasonic_m8eg-pn28080k.dts (hunk 1 above)
Comment (adschm) [reply target/linux/realtek/dts/rtl8380_panasonic_m8eg-pn28080k.dts:64]: Okay, then just using "giga" and "100m" for the node names would be most consistent.

Comment (musashino205) [reply target/linux/realtek/dts/rtl8380_panasonic_m8eg-pn28080k.dts:64]: Thank you, I'll update the names.

**Comment by musashino205** [comment]
UPDATE:

- update the node names in LEDs
  - ```gigabit``` -> ```giga```
  - ```100mbps``` -> ```100m```

**Comment by musashino205** [comment]
UPDATE:

- fix node address of ```vlog_data``` partition
  - ```partition@1e20000``` -> ```partition@1e00000```

**Comment by adschm** [comment]
Sorry, I won't merge anything non-trivial on realtek.

**Comment by musashino205** [comment]
I see, thank you. 
I sympathize with that attitude towards areas where do not have much knowledge.
However, if all the reactions suddenly stop, I may feel that it is left unattended, so it would be helpful if you could leave some mark  :)

**Comment by musashino205** [comment]
I'll update this PR for 5.10 later.

**Comment by musashino205** [comment]
UPDATE:

- fix wrong model name in pca953x commit

**Comment by musashino205** [comment]
I'll re-send the changes to the ML...

**Pull request closed without merging**